spec/acceptance/tests/resource/cron/should_not_overwrite_crontab_file_on_file_read_error_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'spec_helper_acceptance'
```

Comment:
I wonder if the PUP-9217 tests should have an additional confine on `compatible_agents` that selects only those agents whose Puppet versions contain the PUP-9217-modified ParsedFile provider? Otherwise, I think these would fail if any agents in `compatible_agents` correspond to older agents.

Reply:
@joshcooper or @jtappa any thoughts on ^ ?

Reply:
You could mark those tests as pending on the agents that they'd fail on, or you could edit the existing confine to exclude them. I'd consult with the modules team on what their preference or way of handling that would be normally. They might already have a rule or precedent for situations like this.

Reply:
@ekinanp what your doing seems reasonable! 👍 +1